app/assets/javascripts/ga_cross_domain_form_tracking.js

```diff
@@ -0,0 +1,17 @@
+function attachCrossDomainTrackerToInput(form, trackers) {
+  if (trackers.length) {
+    var existingAction = form.attr("action");
```

Comment:
Nitpick: we're moving away from semicolons at the end of lines. Not sure why that hasn't been flagged by the linter.